slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/SecurityConfig.java

```diff
@@ -14,10 +14,9 @@
 	private void applyRouteRestrictions(HttpSecurity http) throws Exception {
 		http.antMatcher("/**").authorizeRequests()
 				// PUBLIC
-				.antMatchers("/auth/**").permitAll()
-				.antMatchers("/swagger-ui/**").permitAll()
-				.antMatchers("/api-docs/**").permitAll()
-				.antMatchers(HttpMethod.GET, "/**").permitAll().anyRequest().authenticated();
+				.antMatchers("/auth/**").permitAll().antMatchers("/swagger-ui/**").permitAll()
+				.antMatchers("/api-docs/**").permitAll().antMatchers(HttpMethod.GET, "/**").permitAll().anyRequest()
+				.authenticated();
```

Comment:
I think you shouln't change this.

Reply:
That was conflict that I resolved in favor of 'develop' branch

Reply:
Compare it with dev branch. It is different.